orcid-model/src/main/java/org/orcid/jaxb/model/message/ScopePathType.java

```diff
@@ -102,7 +102,7 @@
     
     //XXX: Per activity API    
     @XmlEnumValue("/activities/read-limited") ACTIVITIES_READ_LIMITED("/activities/read-limited", ORCID_WORKS_READ_LIMITED, AFFILIATIONS_READ_LIMITED, FUNDING_READ_LIMITED, PEER_REVIEW_READ_LIMITED),
-    @XmlEnumValue("/activities/update") ACTIVITIES_UPDATE ("/activities/update", ACTIVITIES_READ_LIMITED, ORCID_WORKS_CREATE, ORCID_WORKS_UPDATE, AFFILIATIONS_CREATE, AFFILIATIONS_UPDATE, FUNDING_CREATE, FUNDING_UPDATE, PEER_REVIEW_CREATE, PEER_REVIEW_UPDATE),    
+    @XmlEnumValue("/activities/update") ACTIVITIES_UPDATE ("/activities/update", ACTIVITIES_READ_LIMITED, ORCID_WORKS_CREATE, ORCID_WORKS_UPDATE, ORCID_WORKS_READ_LIMITED, AFFILIATIONS_CREATE, AFFILIATIONS_UPDATE, AFFILIATIONS_READ_LIMITED, FUNDING_CREATE, FUNDING_UPDATE, FUNDING_READ_LIMITED, PEER_REVIEW_CREATE, PEER_REVIEW_UPDATE, PEER_REVIEW_READ_LIMITED),    
```

Comment:
@wjrsimpson please note this change, likely conflict with your other work but the order of these scopes matter.